lib/mjsonwp/errors.js

```diff
@@ -1,17 +1,24 @@
 import ES6Error from 'es6-error';
 import _ from 'lodash';
-import { util } from 'appium-support';
+import { util, logger } from 'appium-support';
 import HTTPStatusCodes from 'http-status-codes';
 
+const log = logger.getLogger('MJSONWP');
+
 // base error class for all of our errors
 class MJSONWPError extends ES6Error {
```

Comment:
at some point of time we'll need to properly rename all this stuff

Reply:
Yeah I should make an issue for that. Classes and files will need to be renamed.

Reply:
https://github.com/appium/appium/issues/10041